index.html

```diff
@@ -461,8 +461,8 @@ <h2>This is an interactive API</h2>
                   let n = curve.normal(t);
                   let kr = curve.curvature(t);
                   drawLine(p, {
-                    x: p.x + n.x * kr.k,
-                    y: p.y + n.y * kr.k
+                    x: p.x + n.x * kr.r,
```

Comment:
absolutely not: this graphic is showing off the curvature, not the _radius_ of curvature. The tighter the curve, the larger the curvature value.

Reply:
I replace `kr.k` with `kr.r`, because the results of `n.x * kr.k` are almost less than 1. So there are not any graphic on the most interpolation points .....

Use: `n.x * kr.k`
![image](https://user-images.githubusercontent.com/1681459/54277094-d113f580-45c9-11e9-9abc-f4efed60d3e5.png)

Use: `n.x * kr.r * 10000`
![image](https://user-images.githubusercontent.com/1681459/54277375-a2e2e580-45ca-11e9-986d-f970c3ee3fd4.png)